package.json

```diff
@@ -31,11 +31,11 @@
     "coveralls": "latest",
     "grunt": "~0.4.4",
     "grunt-complexity": "^0.1.7",
-    "grunt-contrib-jshint": "~0.10.0",
+    "grunt-contrib-jshint": "^1.1.0",
```

Comment:
@beatfactor these changes were not a separate topic, it was to pass in Travis using node 6.